.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: objective-c
 
+sudo: false
```

Comment:
depends when it was first added to travis (http://docs.travis-ci.com/user/workers/container-based-infrastructure/#Routing-your-build-to-container-based-infrastructure)

explicit is of course good all the same =-)